training_system/app/controllers/trainer/user_courses_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Trainer::UserCoursesController < ApplicationController
+  before_action :load_course, only: :create
+
+  def new; end
+
+  def create
+    params[:user_ids].each do |user|
+      @course.user_courses.create user_id: user
```

Comment:
tìm hiểu về `transaction in rails `đi bạn ơi. 

cơ mà sao ko dùng build sau đó lấy course save nhỉ